### ISSUE TITLE
`zipcodes()` returns a dataframe with incorrect dtype.

### ISSUE THREAD

**Issue by yy** [altair-viz/vega_datasets#16]
```python
from vega_datasets import data
zipcodes = data.zipcodes()
print(zipcodes.zip_code.dtype)
```

Expected: `dtype('O')` or rather `CategoricalDtype(categories=['00501', '00544', ...`. 

Actual: `dtype('int64')`

Some ZIP codes starts with "0" and `zipcodes = data.zipcodes()` removes all preceding zeros. The following works, but I think it's better to return with the correct dtypes by default. 

```python
zipcodes = data.zipcodes(dtype={'zip_code': 'category'})
```

Also found that `data.unemployment()` cannot correctly parse the data. One should specify the separator `data.unemployment(sep='\t')`.

**Comment by jakevdp** [comment]
Thanks – these can be fixed by creating custom data loading objects in https://github.com/altair-viz/vega_datasets/blob/master/vega_datasets/core.py

If you're interested in submitting a pull request to fix these, I'd be happy to help you get started.

**Comment by jakevdp** [comment]
Fixed in #17 and #18
